### PR TITLE
avillela python manual instrumentation

### PR DESCRIPTION
**Pull request by avillela**
Update Python manual instrumentation example

### CODE REVIEW

File: python/opentelemetry/manual_instrumentation/server.py [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+#   OTEL_RESOURCE_ATTRIBUTES=service.name=py-opentelemetry-manual-otlp-server,service.version=10.10.9
 #   python server.py \
+#
+# Context propagation reference here: https://github.com/open-telemetry/opentelemetry-python/blob/05fd6f3399b1a214c46e71367e124be5d504ad26/opentelemetry-api/src/opentelemetry/propagate/__init__.py#L43-L48
 
 from opentelemetry import trace
-from opentelemetry.instrumentation.flask import FlaskInstrumentor
-from opentelemetry.instrumentation.requests import RequestsInstrumentor
+# from opentelemetry.instrumentation.flask import FlaskInstrumentor
+# from opentelemetry.instrumentation.requests import RequestsInstrumentor
```
Comment (ocelotl) [inline python/opentelemetry/manual_instrumentation/server.py:14]:
```suggestion
```

File: python/opentelemetry/manual_instrumentation/server.py [hunk 2]
```diff
@@ -25,13 +31,17 @@
 # Init tracer
 tracer = get_tracer()
 
-# Init autoinstrumentation with Flask
 app = Flask(__name__)
-FlaskInstrumentor().instrument_app(app)
-RequestsInstrumentor().instrument()
+
+# Init autoinstrumentation with Flask
```
Comment (ocelotl) [inline python/opentelemetry/manual_instrumentation/server.py:36]: same here

File: python/opentelemetry/manual_instrumentation/server.py [hunk 3]
```diff
@@ -99,10 +116,18 @@ def sqlalchemy_integration(length):
 
 @app.route("/rolldice")
```
Comment (ocelotl) [inline python/opentelemetry/manual_instrumentation/server.py:117]: Is `rolldice` still being called?

**Review by ocelotl — APPROVED** [review]

File: python/opentelemetry/manual_instrumentation/server.py (hunk 3 above)
Comment (avillela) [reply python/opentelemetry/manual_instrumentation/server.py:117]: I'll remove.

File: python/opentelemetry/manual_instrumentation/server.py (hunk 2 above)
Comment (avillela) [reply python/opentelemetry/manual_instrumentation/server.py:36]: Removed

File: python/opentelemetry/manual_instrumentation/server.py (hunk 1 above)
Comment (avillela) [reply python/opentelemetry/manual_instrumentation/server.py:14]: Updated

**Pull request merged**